kratos/python_scripts/run_cpp_tests.py

```diff
@@ -1,3 +1,9 @@
-from KratosMultiphysics import *
-Tester.SetVerbosity(Tester.Verbosity.TESTS_OUTPUTS)
-Tester.RunAllTestCases()
+import KratosMultiphysics as Kratos
```

Comment:
This could be an independent PR

Reply:
Yes, but it is useful to run stress tests and check the time